### PR TITLE
Updates default arguments to match current Play example

### PR DESCRIPTION
**Pull request by garanj**
Changes default values for configurable options to align with Play defaults, ensuring users have to do minimum of configuration to achieve the most common settings.

### CODE REVIEW

**Review by alexclarke-g — APPROVED** [review]
Thanks

File: play-validations/memory-footprint/src/main/java/com/google/wear/watchface/dfx/memory/EvaluationSettings.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ final class EvaluationSettings {
 
     private long ambientLimitBytes = MemoryFootprint.toBytes(10);
 
-    private long activeLimitBytes = MemoryFootprint.toBytes(10);
+    private long activeLimitBytes = MemoryFootprint.toBytes(100);
```
Comment (yschimke) [inline play-validations/memory-footprint/src/main/java/com/google/wear/watchface/dfx/memory/EvaluationSettings.java:51]:
```suggestion
    private long activeLimitBytes = MemoryFootprint.toBytes(/* megaBytes = */100);
```

I found this confusing as expressed

Comment (yschimke) [reply play-validations/memory-footprint/src/main/java/com/google/wear/watchface/dfx/memory/EvaluationSettings.java:51]: super nit: I guess it's really mebiByte ?

File: play-validations/memory-footprint/src/test/java/com/google/wear/watchface/dfx/memory/EvaluationSettingsTest.java [hunk 2]
```diff
@@ -14,7 +14,8 @@
 public class EvaluationSettingsTest {
 
     // 10 MB
```
Comment (yschimke) [inline play-validations/memory-footprint/src/test/java/com/google/wear/watchface/dfx/memory/EvaluationSettingsTest.java:16]:
```suggestion
    // 100 MB
```

File: play-validations/memory-footprint/src/test/java/com/google/wear/watchface/dfx/memory/EvaluationSettingsTest.java [hunk 3]
```diff
@@ -14,7 +14,8 @@
...
-    private static final long DEFAULT_MEMORY_LIMIT_BYTES = 10 * 1024 * 1024;
+    private static final long DEFAULT_ACTIVE_MEMORY_LIMIT_BYTES = 100 * 1024 * 1024;
+    private static final long DEFAULT_AMBIENT_MEMORY_LIMIT_BYTES = 10 * 1024 * 1024;
```
Comment (yschimke) [inline play-validations/memory-footprint/src/test/java/com/google/wear/watchface/dfx/memory/EvaluationSettingsTest.java:18]:
```suggestion
    // 10 MB
    private static final long DEFAULT_AMBIENT_MEMORY_LIMIT_BYTES = 10 * 1024 * 1024;
```

**Review by yschimke — APPROVED** [review]

**Pull request merged**